composer.json

```diff
@@ -19,9 +19,16 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.4|^8.0"
+        "php": "^7.4|^8.0",
+        "php-http/httplug": "^2.2",
+        "psr/http-message": "^1.0",
+        "psr/http-server-middleware": "^1.0",
+        "sentry/sentry": "^3.3",
+        "yiisoft/di": "3.0.x-dev"
     },
     "require-dev": {
+        "guzzlehttp/guzzle": "^7.3",
```

Comment:
Why getting both guzzle and http plug at the same time?

Reply:
httpplug needs to configure abstractions in config files. [httplug doesn't provide an adapter](https://github.com/php-http/httplug/tree/master/src).

And yes, I need to move `guzzle` to `require` section too. Are you agree with it or not?

Reply:
I'd prefer not to tie to concrete implementation.